sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/DateTimeUtils.scala

```diff
@@ -455,6 +456,22 @@ object DateTimeUtils {
     (MICROSECONDS.toSeconds(localTimestamp(microsec, timeZone)) % 60).toInt
   }
 
+  /**
+   * Returns seconds, including fractional parts, multiplied by 1 000. The timestamp
+   * is expressed in microseconds since the epoch.
+   */
+  def getMilliseconds(timestamp: SQLTimestamp, timeZone: TimeZone): Int = {
+    getMicroseconds(timestamp, timeZone) / MICROS_PER_MILLIS.toInt
+  }
+
+  /**
+   * Returns seconds, including fractional parts, multiplied by 1 000 000. The timestamp
```

Comment:
`1 000 000` -> `1000000`.

Reply:
The same question. I believe `1 000 000` looks better.